README.md

```diff
@@ -29,9 +29,10 @@
  - RESTBot and GatewayBot support.
 
 ### Links
+> 📦 | [Pypi](https://pypi.org/project/hikari-crescent/)<br>
 > 📖 | [User Guide](https://hikari-crescent.github.io/book)<br>
-> 🗃️ | [Docs](https://hikari-crescent.github.io/hikari-crescent/crescent.html)<br>
```

Comment:
What about the book?

Reply:
oh it's API references now?

Reply:
The "User Guide" is the book. I changed docs to API reference so there was less ambiguity between what the user guide and api reference where.